hw3/hw3.py

```diff
@@ -0,0 +1,352 @@
+"""This is sollution for homework 3.
+
+Медиатека.
+Опишите архитектуру классов для управления медиатекой (коллекцией мультимедийных файлов).
+Вам нужно создать следующие классы:
+
+Класс "Медиафайл":
+Поля:
+Название файла
+Расширение файла
+Методы:
+Геттеры и сеттеры для полей
+
+Класс "Аудиофайл" (наследуется от "Медиафайла"):
+Дополнительные поля:
+Исполнитель
+Длительность
+Методы:
+Геттеры и сеттеры для дополнительных полей
+
+Класс "Видеофайл" (наследуется от "Медиафайла"):
+Дополнительные поля:
+Разрешение
+Продолжительность
+Методы:
+Геттеры и сеттеры для дополнительных полей
+
+Класс "Медиатека":
+Поля:
+Список медиафайлов (массив или список объектов класса "Медиафайл")
+Методы:
+Добавить медиафайл в медиатеку
+Удалить медиафайл из медиатеки
+Получить список всех медиафайлов в медиатеке.
+"""
+from abc import ABC, abstractmethod
+
+VALID_FILETYPES = ('.mp4', '.wav', '.mov', '.jpeg', '.jpg', '.png', '.svg', '.raw', '.mp3')
```

Comment:
why having a dot in every file type if you already know that this is a file type :)

a dot should be added implicitly when you are concatenating a name and a type

Reply:
Due to some weird reason I can't in setter add dot to it... ```'.' + new_filetype```  just doesn't work. Maybe it can be done in __init__, but still some wierd issue.